cabal-install/Distribution/Client/CmdRepl.hs

```diff
@@ -125,9 +125,9 @@ defaultEnvFlags = EnvFlags
 envOptions :: ShowOrParseArgs -> [OptionField EnvFlags]
 envOptions _ =
   [ option ['b'] ["build-depends"]
-    "Include an additional package in the environment presented to GHCi."
+    "Include additional packages in the environment presented to GHCi."
     envPackages (\p flags -> flags { envPackages = p ++ envPackages flags })
-    (reqArg "DEPENDENCY" dependencyReadE (fmap prettyShow :: [Dependency] -> [String]))
+    (reqArg "DEPENDENCIES" dependencyReadE (fmap prettyShow :: [Dependency] -> [String]))
```

Comment:
`dependenciesReadE`

Also change error from `couldn't parse dependency:` to `couldn't parse dependencies:`
```

Reply:
Thanks @phadej, good call